City-of-Helsinki/drupal-hdbt
# UHF-10757

# [UHF-10757](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10757)
Fixed a bunch of javascript errors from sentry;
- UHF-10755: SecurityError (localstorage usage on incognito might cause this error)
- UHF-10757: ResizeObserver and UnhandeldRejection (caused  by third party most likely. [UnhandledRejection errors](https://forum.sentry.io/t/unhandledrejection-non-error-promise-rejection-captured-with-value/14062/40) might be caused by outlook safelink crawler ) 
- UHF-9969 Finally remove the localstorage cleaner from the codebase. 

Also, running npm build update all of the minified react codes.

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-10757`
* Run `make drush-cr`

## How to test
- Test that localstorage is still working (accordion for example)
- Browse the site a bit and javascript features should work as expected
- Test one or more of the react-searches



[UHF-10757]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10757?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
